font-patcher

```diff
@@ -817,6 +842,20 @@ class font_patcher:
             # does not overlap the bearings (edges)
             self.remove_glyph_neg_bearings(self.sourceFont[currentSourceFontGlyph])
 
+            # Check if the inserted glyph is scaled correctly for monospace
+            if self.args.single:
+                (xmin, _, xmax, _) = self.sourceFont[currentSourceFontGlyph].boundingBox()
+                if int(xmax - xmin) > self.font_dim['width'] * (1 + overlap):
+                    print("  Warning: Scaled glyph U+{:X} wider than one monospace width ({} / {} (overlap {}))".format(
+                        currentSourceFontGlyph, int(xmax - xmin), self.font_dim['width'], overlap))
+
+            # reset selection so iteration works properly @TODO fix? rookie misunderstanding?
+            # This is likely needed because the selection was changed when the glyph was copy/pasted
+            if symbolFontStart == 0:
+                symbolFont.selection.all()
+            else:
+                symbolFont.selection.select((str("ranges"), str("unicode")), symbolFontStart, symbolFontEnd)
+
```

Comment:
This can be removed how with #736 ?

Reply:
This is a merge error. It has not been in any of my commits.
I usually do a rebase on master instead of merging in master; find that easier to follow and later on it is not so much work to pull it (fast forward will be possible).

I will drop your merge commit and do a proper rebase, it should be vanished then.